presto-main/src/main/java/com/facebook/presto/sql/planner/LocalExecutionPlanner.java

```diff
@@ -588,6 +588,9 @@ public PhysicalOperation visitRemoteSource(RemoteSourceNode node, LocalExecution
         {
             List<Type> types = getSourceOperatorTypes(node, context.getTypes());
 
+            if (node.isOrderSensitive()) {
```

Comment:
Could you give an example of a plan where it is needed? I think that `OutputNode` is executed with just one driver. Now you added additional exchange just below `OutputNode`. What would happen if you didn't set `context.setDriverInstanceCount(1);`? Would `OutputNode` be executed with more than one driver (it doesn't make sense, right)? How does the plan look like?